kolena/_utils/batched_load.py

```diff
@@ -128,7 +128,7 @@ def iter_data(
         init_request: API.BaseInitDownloadRequest,
         endpoint_path: str,
         df_class: Optional[Type[DFType]],
-        endpoint_api_version: int = DEFAULT_API_VERSION,
+        endpoint_api_version: str = DEFAULT_API_VERSION,
     ) -> Iterator[DFType]:
         kreq = krequests if endpoint_api_version == API_V1 else krequests_v2
         with kreq.put(
```

Comment:
mypy has issue with module as variable, which they don't seem to intend to support. Unfortunately have to `#type: ignore` on this kind of usage in this file.